spec/components/date_range_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe 'date_range', type: :helper do
+  subject do
+    Nokogiri::HTML.parse(date_range)
+  end
+
+  let(:form) do
+    helper.bootstrap_form_for :foo, url: '/' do |f|
+      return f
```

Comment:
I guess it's returning from wherever this is evaluated and nothing crucial happens after that point in that method... So, it's kind of assuming this, which is not that nice. On the other hand it's a test, so it is tested behaviour. ;)